src/api/spec/controllers/webui/attribute_controller_spec.rb

```diff
@@ -174,10 +174,11 @@
   end
 
   describe 'DELETE #destroy' do
-    let!(:attrib) { create(:attrib, project: user.home_project) }
+    let(:attrib) { create(:attrib, project: user.home_project) }
 
     before do
       login user
+      attrib
```

Comment:
I don't think this is needed when `attrib` is lazy-evaluated below.

Reply:
I find it strange to rely on it being lazy created when we want to test it's deleted. But I can take it out

Reply:
also true...